src/mame/drivers/magic10.cpp

```diff
@@ -782,6 +807,9 @@ void magic10_state::magic102(machine_config &config)
 	m_maincpu->set_addrmap(AS_PROGRAM, &magic10_state::magic102_map);
 
 	subdevice<screen_device>("screen")->set_visarea(0*8, 48*8-1, 0*8, 30*8-1);
+
+	config.device_remove("ticket");
+	config.device_remove("hopper");
```

Comment:
Please build machines additively where possible – prefer using common base configuration method, and then adding devices for machines that have them, rather than removing devices in derived configurations.

Reply:
I can do this, but can't solve about earlier comment.